src/panels/config/integrations/integration-panels/mqtt/mqtt-subscribe-card.ts

```diff
@@ -47,32 +47,35 @@ class MqttSubscribeCard extends LitElement {
         header=${this.hass.localize("ui.panel.config.mqtt.description_listen")}
       >
         <form>
-          <ha-textfield
-            .label=${this._subscribed
-              ? this.hass.localize("ui.panel.config.mqtt.listening_to")
-              : this.hass.localize("ui.panel.config.mqtt.subscribe_to")}
-            .disabled=${this._subscribed !== undefined}
-            .value=${this._topic}
-            @change=${this._handleTopic}
-          ></ha-textfield>
-          <ha-select
-            .label=${this.hass.localize("ui.panel.config.mqtt.qos")}
-            .disabled=${this._subscribed !== undefined}
-            .value=${this._qos}
-            @selected=${this._handleQos}
-            >${qosLevel.map(
-              (qos) => html`<mwc-list-item .value=${qos}>${qos}</mwc-list-item>`
-            )}
-          </ha-select>
-          <mwc-button
-            .disabled=${this._topic === ""}
-            @click=${this._handleSubmit}
-            type="submit"
-          >
-            ${this._subscribed
-              ? this.hass.localize("ui.panel.config.mqtt.stop_listening")
-              : this.hass.localize("ui.panel.config.mqtt.start_listening")}
-          </mwc-button>
+          <div class="panel-dev-mqtt-subsribe-fields">
```

Comment:
```suggestion
          <div class="panel-dev-mqtt-subscribe-fields">
```

Reply:
No need to be so specific with the class names btw, it is a shadow root, so isolated :-)